### ISSUE TITLE
Any plans to get this working in Visual Studio for Mac?

### ISSUE THREAD

**Issue by roydukkey** [rubicon-oss/LicenseHeaderManager#99]
I guess extensions are different for VS for Mac. Are you able to develop a version for Mac?

**Comment by MichaelKetting** [comment]
Yes, according to a quick google search they appear to be entirely different from Windows-based Visual Studio.

At this point there's no plans to do extensions for Mac.

**Comment by roydukkey** [comment]
Would you accept pull requests, should someone be willing to research and provide a solution? Or, would you prefer an equivalent Mac extension remain separate?

**Comment by MichaelKetting** [comment]
That's currently being discussed with management. Will get back to you in a couple of days.

**Comment by stefan-wenig** [comment]
We would discuss a proposal to provide a pull request that does the following 
- refactor the code so that the core functionality is separated from code with EnvDTE dependencies (the .NET classes exposed to VS/Win add-ins)
- package the core functionality as a nuget package

If you do this, the core functions can be shared between the VS/Win add-in and the VS/Mac add-in.

We cannot maintain the VS/Mac version.
